var/spack/repos/builtin/packages/ploticus/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Ploticus(MakefilePackage):
+    """Ploticus can produce various types of plots and graphs."""
+
+    homepage = "http://ploticus.sourceforge.net/doc/welcome.html"
+
+    maintainers("Christoph-TU")
+
+    version("2.42", sha256="3f29e4b9f405203a93efec900e5816d9e1b4381821881e241c08cab7dd66e0b0")
+
+    depends_on("zlib")
```

Comment:
```suggestion
    depends_on("zlib-api")
```

Reply:
@Christoph-TU Would it work with this change :point_up: ? Lately we switched from `zlib` to `zlib-ng` by default, and added `zlib-api` as a virtual.

Reply:
Applied. Yes, it builds with zlib-ng.